base/applications/regedit/lang/ro-RO.rc

```diff
@@ -31,17 +31,17 @@ Opțiuni:\n\
                  Compatibil cu orice alt comutator. Ignorat.\n\
   /R:user.dat    Locația fișierului user.dat urmează să fie modificată.\n\
                  Compatibil cu orice alt comutator. Ignorat.\n\
-  /C             Importă conținutul unui fișier de regiștri.\n\
-  /D             Șterge o cheie de regiștrii specificată.\n\
-  /E             Exportă conținutul unei cheie de regiștri specificată într-un fișier.\n\
+  /C             Importă conținutul unui fișier de regiștri. Este importat întregul registru dintr-un fișier.\n\
```

Comment:
This addition doesn't make sense. (My previous comment was a suggestion only for the line №37, to which it was attached to.) Here, what you should have done is only to change "regiștri" to "registru".